test/suites/integration/httproute_method_match_test.go

```diff
@@ -0,0 +1,124 @@
+package integration
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-application-networking-k8s/test/pkg/test"
+	"github.com/aws/aws-sdk-go/service/vpclattice"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("HTTPRoute method matches", func() {
+	It("HTTPRoute should route by HTTP method", func() {
+		gateway := testFramework.NewGateway("", k8snamespace)
+		deployment1, getService := testFramework.NewHttpApp(test.HTTPAppOptions{Name: "test-get", Namespace: k8snamespace})
+		deployment2, postService := testFramework.NewHttpApp(test.HTTPAppOptions{Name: "test-post", Namespace: k8snamespace})
+		methodMatchHttpRoute := testFramework.NewMethodMatchHttpRoute(gateway, getService, postService, "route-by-method", k8snamespace)
+
+		// Create Kubernetes API Objects
+		testFramework.ExpectCreated(ctx,
+			gateway,
+			methodMatchHttpRoute,
+			getService,
+			deployment1,
+			postService,
+			deployment2,
+		)
+
+		time.Sleep(3 * time.Minute)
```

Comment:
Why do we sleep here?

Reply:
Every routing test does this, I'm just following the crowd. Agree this isn't ideal and shouldn't be our approach as it could artificially extend the time the test takes to run.